addon/styles/addon.scss

```diff
@@ -40,47 +44,79 @@
 }
 
 .frost-list {
-	display: flex;
-	flex-direction: column;
-}
-
-.frost-list .container::before,
-.frost-list .container::after {
-	content: '';
-	position: absolute;
-	top: 0;
-	width: 1px;
-	height: 100%;
-	background-color: $frost-color-lgrey-1;
-}
-
-.frost-list .container::before {
-	left: 0;
-}
-
-.frost-list .container::after {
-  right: 0;
-}
-
-.frost-list .border-top {
-	height: 1px;
-  min-height: 1px;
-	margin-right: 15px;
-	background-color: $frost-color-lgrey-1;
-}
+  display: flex;
+  flex-direction: column;
+
+  .content {
+    height: calc(100vh - #{$frost-list-header-height});
+
+    .border-top {
+      height: $frost-list-content-border-top;
+      min-height: 1px;
+      margin-right: 15px;
+      background-color: $frost-color-lgrey-1;
+    }
+    .container {
+      height: calc(100% - #{$frost-list-content-border-top + $frost-list-content-border-bottom});
+      border-right: 1px solid $frost-color-lgrey-1;
+      border-left: 1px solid $frost-color-lgrey-1;
+      background-color: $frost-color-white;
+
+      vertical-collection {
+        border-bottom: 1px solid $frost-color-lgrey-1
+      }
+    }
+    .border-bottom {
+      height: $frost-list-content-border-bottom;
+      min-height: 1px;
+      margin-right: 15px;
+      background-color: $frost-color-lgrey-1;
+    }
+  }
 
-.frost-list .container {
-	border-right: 1px solid $frost-color-lgrey-1;
-	border-left: 1px solid $frost-color-lgrey-1;
-	background-color: $frost-color-white;
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    height: $frost-list-header-height;
+
+    .frost-list-toggle {
+      display: flex;
+      position: relative;
+      flex-direction: row;
+      align-items: center;
+      margin: 0 15px 0 0;
+      font-size: $frost-font-s;
+      cursor: pointer;
+
+      .frost-list-toggle-item {
+        height: 25px;
+        line-height: 25px;
+        padding: 0 10px 0 10px;
+        color: $frost-color-blue-1;
+
+        &:hover {
+          background-color: rgba($frost-color-blue-1, 0.05);
+        }
+
+        &:nth-child(n + 2) {
```

Comment:
This is the divider between Expand all and Collapse all?  Why do we need the nth selector?